bigtable/client/client_options.h

```diff
@@ -53,6 +53,126 @@ class ClientOptions {
     return *this;
   }
 
+  /**
+   * Set Compression Algorithm for channel.
```

Comment:
"compression algorithm" should be lowercase here, it's not a proper name but just used descriptively.

Reply:
Done.